app_dart/lib/src/request_handlers/scheduler/batch_backfiller.dart

```diff
@@ -50,14 +50,28 @@ class BatchBackfiller extends RequestHandler {
     return Body.empty;
   }
 
+  // Remove the ci_yaml rollers from backfill list.
+  List<FullTask> removeCiYamlRollerFromBackFill(List<FullTask> taskList) {
```

Comment:
Can we control from .ci.yaml to make it easy to enable/disable? Say adding a property `backfill: false` to skip ones that we do not want to backfill.

Reply:
Yeah I guess so. Let me look into that. Do I need to then update a proto somewhere?

Reply:
No. Properties is already supported as in proto, and here we just add a sub element to the properties.

Reply:
Ah okay. Yeah I will do that then.

Reply:
Okay that is a much better solution. Implemented this change.